moveos/moveos/src/vm/moveos_vm.rs

```diff
@@ -187,7 +192,7 @@ where
 
                 let mut init_function_modules = vec![];
                 for module in &compiled_modules {
-                    let result = moveos_verifier::verifier::verify_module(module);
+                    let result = moveos_verifier::verifier::verify_module(module, db);
```

Comment:
Just use `&self.remote` to replace `db` at here.

Reply:
Okay, I have fixed it.